tests/db_functions/math/test_log.py

```diff
@@ -0,0 +1,39 @@
+import math
+from decimal import Decimal
+
+from django.db.models.functions import Log
+from django.test import TestCase
+
+from ..models import DecimalModel, FloatModel, IntegerModel
+
+
+class LogTests(TestCase):
+
+    def assertLogEqual(self, n_log, n1, n2):
```

Comment:
Please remove this again and change back to `assertAlmostEqual`.

I know that it causes a test failure at the moment, but we need to fix it properly. This assertion method just makes the assumption that passing in one way or the other works. We have already ascertained that all backends that define the function use `LOG(B, X)`, and Python has `math.log(x, b)` which is already catered for in the function creation for `sqlite`.

The issue here is that `sqlite` is working and `spatialite` is failing with the arguments in the wrong order. This is strange as `spatialite` either ought to use the same function as `sqlite` or what looks like it's own function which ought to be compatible: [`LOG(B, X)`](http://www.gaia-gis.it/gaia-sins/spatialite-sql-4.3.0.html#math).